src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -9,8 +9,14 @@ public class Hardware {
 
 	// climb [20 - 29]
 
-	// shooter [30 - 39]
-
+	// launcher [30 - 39]
+	public static final int LAUNCHER_TOP_MOTOR_ID = 30;
+	public static final int LAUNCHER_BOTTOM_MOTOR_ID = 31;
+	public static final int LAUNCHER_ANGLE_MOTOR_ID = 32;
+	public static final int LAUNCHER_HOOD_MOTOR_ID = 33;
 	// intake [40 - 49]
 
+	// sensors [1 - 19]
+	public static final int LAUNCHER_SENSOR_1 = 1;
+	public static final int LAUNCHER_SENSOR_2 = 2;
```

Comment:
It'd be better to name these based on where they are on the robot

Reply:
Yeah, we will change it once we have a idea on every sensor on the bot and number it accordingly, ty!